### PR TITLE
feat: add new LSB navigation components and router item state management

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Self-reviewed (coding conventions, bug-free, functionality verified, tests checked, documentation updated)
- [ ] Minor change, review optional (`style`, `chore`, `ci`, straightforward changes, etc.)
- [ ] Previously reviewed in feature branch, no further review needed
- [x] Need review or discussion

####  Example
image

```html
<l-s-b-container>
    <l-s-b-router-button icon="ic_dots-4-square">
        All Categories
    </l-s-b-router-button>
    <l-s-b-divider />
    <l-s-b-top-title>Category</l-s-b-top-title>
    <l-s-b-router-item id="test"
                       to="/test"
    >
        Test
    </l-s-b-router-item>
</l-s-b-container>
```

#### Key Changes

1. **New LSB Navigation Components Added**
   - **`LSBContainer`**: Base container for padding
   - **`LSBDivider`**: Divider between items
   - **`LSBTopTitle`**: Displays the top title of the LSB
   - **`LSBRouterButton`**: Single router button
   - **`LSBRouterItem`**: Repeatedly rendered router item

2. **LSB Router Item State Hook Added**
   - Common hook to manage routing state (current path, active status, etc.)
 
---
  
#### 주요 변경사항

1. **새로운 LSB 내비게이션 컴포넌트 추가**
   - **`LSBContainer`**: padding 처리용 기본 컨테이너
   - **`LSBDivider`**: 항목 간 구분선
   - **`LSBTopTitle`**: LSB 상단 제목 표시
   - **`LSBRouterButton`**: 단일 라우터 버튼
   - **`LSBRouterItem`**: 반복 렌더링되는 라우터 항목

2. **LSB 라우터 아이템 상태 훅 추가**
   - 라우팅 상태(현재 경로, 활성 여부 등)를 관리하는 공통 훅 구현

### CODE REVIEW

File: apps/web/src/common/modules/navigations/new-lsb/composables/use-lsb-router-item-state.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    });
+    const imgIconUrl = computed<string>(() => (props.imgIcon ? assetUrlConverter(props.imgIcon) : ''));
+
+    return {
+        isSelected,
+        iconName,
+        iconColor,
+        imgIconUrl,
+    };
+};
```
Comment (sulmoJ) [inline apps/web/src/common/modules/navigations/new-lsb/composables/use-lsb-router-item-state.ts:50]: 👍

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Review by skdud4659 — APPROVED** [review]
LGTM!

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Comment by WANZARGEN** [comment]
I applied pinia minor version upgrade:

1. Bug Fixes
storeToRefs Issues

Fixed reactivity breaking after Hot Module Replacement (HMR).
Improved type definitions for nested ref usage.
Allowed writable getters by updating type definitions.
Resolved reactivity issues when using storeToRefs.
Type Definition Fixes

Improved handling of union types in generic parameters for defineStore.
Prevented setup stores from having their types unwrapped twice.
Fixed issues with type definitions when using auto-imported components.
Other Bugs

Ensured patch method expects unwrapped states.
Retained side-effect-free comments during code minification.
Prevented memory leaks in Server-Side Rendering (SSR) environments.
2. Features and Improvements
Plugin System

Introduced a new plugin system to enhance extensibility.
Tree Shaking

Improved tree-shaking functionality in defineStore to reduce bundle size.
3. Miscellaneous
Updated the minimum peer dependency version for Vue.
Improved developer experience with better HMR handling.

**Pull request merged**